sark-localbuild

```diff
@@ -189,6 +191,10 @@ if [ ${REPO_CLEAN_EXPIRE} -eq 1 ] ; then
   eit vacuum --quick --days ${REPO_CLEAN_EXPIRE_DAYS} ${REPOSITORY_NAME}
 fi
 
+if [ x${REPO_BINHOST_CLEAN} == "x1" ] ; then
```

Comment:
yay syntax from 1970s or so; use [[ ]] without x

Reply:
Nooo, this means that I'm very old. :)

Reply:
yet you write in Go as well :D